src/Extensions/AzureBlobStorage/Program.cs

```diff
@@ -53,7 +49,14 @@ private static async Task<int> Egress(string ProviderName)
 
                 Console.CancelKeyPress += Console_CancelKeyPress;
 
-                result.ArtifactPath = await provider.EgressAsync(Constants.AzureBlobStorageProviderName, configPayload.ProviderName, options, GetStream, configPayload.Settings, CancelSource.Token);
+                result.ArtifactPath = await provider.EgressAsync(options,
+                    async (stream, token) =>
```

Comment:
This can be simplified by just passing `GetStream` instead of creating an anonymous async function that merely forwards the arguments.